test/helpers/lsp.cc

```diff
@@ -10,23 +10,21 @@
 namespace sorbet::test {
 using namespace std;
 
-string filePathToUri(string_view prefixUrl, string_view filePath) {
-    return fmt::format("{}/{}", prefixUrl, filePath);
+string filePathToUri(const LSPConfiguration &config, string_view filePath) {
```

Comment:
Below, I've removed all `prefixUrl` arguments in favor of using `LSPConfiguration` directly.